src/main/java/edu/utdallas/davisbase/catalog/DavisBaseTablesTableColumn.java

```diff
@@ -9,17 +9,21 @@
  * catalog table.
  */
 public enum DavisBaseTablesTableColumn implements CatalogTableColumn {
-  ROWID      (DataType.INT,  false),
-  TABLE_NAME (DataType.TEXT, false);
+  ROWID      (DataType.INT,  false, true, false),
+  TABLE_NAME (DataType.TEXT, false, false, false);
 
   private final DataType dataType;
   private final boolean isNullable;
+  private final boolean isUnique;
+  private final boolean isPrimaryKey;
 
-  private DavisBaseTablesTableColumn(DataType dataType, boolean isNullable) {
+  private DavisBaseTablesTableColumn(DataType dataType, boolean isNullable, boolean isUnique, boolean isPrimaryKey) {
     assert dataType != null : "dataType should not be null";
 
```

Comment:
```suggestion
    assert !isPrimaryKey || (isNullable && isUnique);

```

Just a sanity check in case we mistype one of the literal values up above.